pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -316,11 +316,14 @@ func setDeploymentStatus(instance *api.KieApp, resources []resource.KubernetesRe
 
 func (reconciler *Reconciler) verifyExternalReferences(cr *api.KieApp) error {
 	var err error
+
 	if cr.Status.Applied.Auth != nil && cr.Status.Applied.Auth.RoleMapper != nil {
 		err = reconciler.verifyExternalReference(cr.GetNamespace(), cr.Status.Applied.Auth.RoleMapper.From)
 	}
-	if err == nil && cr.Status.Applied.Objects.Console.GitHooks != nil {
-		err = reconciler.verifyExternalReference(cr.GetNamespace(), cr.Status.Applied.Objects.Console.GitHooks.From)
+	if cr.Spec.Objects.Console != nil {
```

Comment:
should be `cr.Status.Applied.Objects.Console`

Reply:
applied.